operations/finance.md

```diff
@@ -267,8 +270,9 @@ Returns all accounting items of the enterprise that were consumed \(posted\) or
 | `AccessToken` | string | required | Access token of the client application. |
 | `Client` | string | required | Name and version of the client application. |
 | `TimeFilter` | string [Accounting item time filter](finance.md#accounting-item-time-filter) | optional | Time filter of the interval. If not specified, items `Consumed` in the interval are returned. |
-| `StartUtc` | string | required | Start of the interval in UTC timezone in ISO 8601 format. |
-| `EndUtc` | string | required | End of the interval in UTC timezone in ISO 8601 format. |
+| `StartUtc` | string | optional | Start of the interval in UTC timezone in ISO 8601 format. Required if `ItemIds` are not specified. |
+| `EndUtc` | string | optional | End of the interval in UTC timezone in ISO 8601 format. Required if `ItemIds` are not specified.  |
+| `ItemIds` | array of string | optional | Unique identifiers of the Accounting items. Required if `StartUtc` and `EndUtc` are not specified. |
```

Comment:
Did you discuss with guys, if it makes sense to write here about the fallback for states?

Reply:
It's already described on line 278

Reply:
nit: I'd go with lowercase a in accounting items.